backend/routes/geo.js

```diff
@@ -5,10 +5,10 @@ const countries = require("../data/countries");
 const { validateGeolocation } = require("../validation/geolocation");
 
 const router = express.Router();
-const GEO_SERVICE_URL = "http://127.0.0.1:5000/geo-service";
+const GEO_SERVICE_URL = "http://0.0.0.0:5000/geo-service";
```

Comment:
Can we move GEO_SERVICE_URL to an environment variable?

Reply:
yes, definitely good suggestion